<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Azael Avalos &lt;coproscefalo@gmail.com&gt;</author>
 <class>SysInfo</class>
 <widget class="QWidget" name="SysInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>General</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="flat">
      <bool>true</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" alignment="Qt::AlignLeft|Qt::AlignVCenter">
       <widget class="QLabel" name="model_family_label">
        <property name="text">
         <string>Model family:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <spacer name="model_family_spacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="2" alignment="Qt::AlignRight|Qt::AlignVCenter">
       <widget class="QLabel" name="model_family">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0" alignment="Qt::AlignLeft|Qt::AlignVCenter">
       <widget class="QLabel" name="model_number_label">
        <property name="text">
         <string>Model number:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <spacer name="model_number_spacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="2" alignment="Qt::AlignRight|Qt::AlignVCenter">
       <widget class="QLabel" name="model_number">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0" alignment="Qt::AlignLeft|Qt::AlignVCenter">
       <widget class="QLabel" name="bios_version_label">
        <property name="text">
         <string>BIOS version:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <spacer name="bios_version_spacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="2" alignment="Qt::AlignRight|Qt::AlignVCenter">
       <widget class="QLabel" name="bios_version">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="0" alignment="Qt::AlignLeft|Qt::AlignVCenter">
       <widget class="QLabel" name="bios_date_label">
        <property name="text">
         <string>BIOS date:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <spacer name="bios_date_spacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="2" alignment="Qt::AlignRight|Qt::AlignVCenter">
       <widget class="QLabel" name="bios_date">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="0" alignment="Qt::AlignLeft|Qt::AlignVCenter">
       <widget class="QLabel" name="bios_manufacturer_label">
        <property name="text">
         <string>BIOS manufacturer:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <spacer name="bios_manufacturer_spacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="2" alignment="Qt::AlignRight|Qt::AlignVCenter">
       <widget class="QLabel" name="bios_manufacturer">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="5" column="0" alignment="Qt::AlignLeft|Qt::AlignVCenter">
       <widget class="QLabel" name="ec_version_label">
        <property name="text">
         <string>EC version:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <spacer name="ec_version_spacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="5" column="2" alignment="Qt::AlignRight|Qt::AlignVCenter">
       <widget class="QLabel" name="ec_version">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="6" column="0" alignment="Qt::AlignLeft|Qt::AlignVCenter">
       <widget class="QLabel" name="driver_version_label">
        <property name="text">
         <string>Driver version:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <spacer name="driver_version_spacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="6" column="2" alignment="Qt::AlignRight|Qt::AlignVCenter">
       <widget class="QLabel" name="driver_version">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
